execution/executor/tests/db_bootstrapper_test.rs

```diff
@@ -97,9 +98,9 @@ fn execute_and_commit(txns: Vec<Transaction>, db: &DbReaderWriter) {
 
 fn get_demo_accounts() -> (
     AccountAddress,
-    AuthenticationKey,
+    Ed25519PrivateKey,
```

Comment:
dumb question: what's the difference here

Reply:
the transfer txns introduced in this PR need to be signed by the private key, while the existing mint transactions only needed the auth key, and the auth key is derivable from the private key so exposing the private key.